getschema/impl.py

```diff
@@ -323,7 +323,7 @@ def fix_type(obj, schema, dict_path=[], on_invalid_property="raise",
         for key in keys:
             ret = fix_type(obj[key], schema, dict_path + ["properties", key],
                            on_invalid_property)
-            if ret:
+            if ret is not None:
```

Comment:
Can you do the same thing at Ln 337 (in array type)? Thx!

Reply:
Done!